src/sqs_adaptor.py

```diff
@@ -57,6 +57,10 @@ def __init__(self, queue_name):
 
     def write(self, string):
         "called when given a validated response"
+        # totally naive. unicode check probably needed
+        if len(string) >= 256000:
+            LOG.error("message to be sent is very large and will be truncated to 256KB")
+        string = string[:256000]
```

Comment:
given this is usually a JSON document, truncating it like this would make it unparseable so not very useful on the other side? We could try to go through the fields and truncate those instead, but I suspect it's easier to go to the source and truncate the validation error message when it is produced, even if it introduces some knowledge of SQS into that part